text/dotnet-core/rfcs/0004-extend-dotnet-sdk.md

```diff
@@ -0,0 +1,50 @@
+# Extend .Net SDK Dependency
+
+## Proposal
+
+Reconfigure the .Net SDK dependency to be identical to the one that can be downloaded from Microsoft which includes the runtimes that the SDK is dependent on to run.
+
+## Motivation
+As of .Net SDK version ~6.0.100 publish now longer properly functions for certain types of .Net apps. Upon testing we were able to get the buildpack to function again if we used an .Net SDK dependency that still had the runtimes libraries packaged with it. After further investigation we discovered that the removal of those local runtimes was what was causing the failure, leading us to believe that the .Net SDK no longer follows the runtime search path that it used to.
```

Comment:
nit: can you link out to the issue that has a lot of details about the failure? https://github.com/paketo-buildpacks/dotnet-core/issues/670